modules/virt-customizing-storage-profile.adoc

```diff
@@ -108,3 +108,24 @@ status:
 <1> Specify the access mode.
 <2> Specify the volume mode.
 <3> Specify the default cloning strategy.
+
+[id="virt-customizing-storage-profile-providers-and-default-behaviors_{context}"]
+=== Storage Providers and Default Behaviors
+
+[cols="1,1",options="header"]
+|===
+|Storage Provider|Default Behavior
```

Comment:
Should be sentence case per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings) (section titles and table headings):
```suggestion
=== Storage providers and default behaviors

[cols="1,1",options="header"]
|===
|Storage provider|Default behavior
```